amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
+
+-- Create new tables
+
+CREATE TABLE extraction_kit_lot (
+	extraction_kit_lot_id bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_extraction_kit_lot PRIMARY KEY ( extraction_kit_lot_id ),
+	CONSTRAINT uq_extraction_kit_lot_name UNIQUE ( name )
+ );
+
+CREATE TABLE extraction_robot (
+	extraction_robot_id  bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_extraction_robot PRIMARY KEY ( extraction_robot_id ),
+	CONSTRAINT uq_extraction_robot_name UNIQUE ( name )
+ );
+
+CREATE TABLE master_mix_lot (
+	master_mix_lot_id    bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_master_mix_lot PRIMARY KEY ( master_mix_lot_id ),
+	CONSTRAINT uq_master_mix_lot_name UNIQUE ( name )
+ );
+
+CREATE TABLE plate_type (
+	plate_type_id        bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	cols                 smallint  NOT NULL,
+	rows                 smallint  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_plate_type PRIMARY KEY ( plate_type_id ),
+	CONSTRAINT uq_plate_type_name UNIQUE ( name )
+ );
+
+CREATE TABLE processing_robot (
+	processing_robot_id  bigserial  NOT NULL,
+	name                 varchar  NOT NULL,
+	description          varchar  ,
+	CONSTRAINT pk_processing_robot PRIMARY KEY ( processing_robot_id ),
+	CONSTRAINT uq_processing_robot_name UNIQUE ( name )
+ );
+
+CREATE TABLE run (
+	run_id               bigserial  NOT NULL,
+	run_date             date  ,
+	description          varchar  ,
+	CONSTRAINT pk_run PRIMARY KEY ( run_id )
+ );
+
+CREATE TABLE sample (
+	sample_id            varchar  NOT NULL,
+	barcode              varchar  ,
+	description          varchar  ,
+	CONSTRAINT pk_sample PRIMARY KEY ( sample_id ),
+	CONSTRAINT fk_sample_barcode FOREIGN KEY ( barcode ) REFERENCES barcodes.barcode( barcode )
+ );
+CREATE INDEX idx_sample_barcode ON sample ( barcode );
+
+CREATE TABLE study (
+	study_id             bigint  NOT NULL,
+	description          varchar  ,
```

Comment:
Instead of "description" add a column named "title" with a unique constraint. This column will automatically be populated with the Qiita information if the study already exists in there.

I think it will also be useful to add "study_alias" as it will be a shorter name that it will be easier for the lab to find it. It is not ensured to be unique, but it will have the same info as in Qiita, if already exists in there.